newsparser.py

```diff
@@ -34,14 +34,31 @@ def __init__(self, author, topic, subject, date, raw_msg, raw_html, mention_mana
         self.author_displayname = author[1]
         self.topic = topic
         self.subject = subject
-        self.date = date
+        self.date = self.getHumanReadableDate(date)
         self.raw_msg = raw_msg
         self.raw_html = raw_html
         self.mention_manager = mention_manager
         self.broken = False
         self.content = None
         self.is_plus_one = None
 
+    def getHumanReadableDate(self, date):
```

Comment:
can you make this a free function instead of a method ? as it doesn't depend on internal state of the article.

Reply:
Sure, I will place the function under the same file, near `isPlusOne`.